LoopUITests/Screens/SystemSettingsScreen.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  SystemSettingsScreen.swift
+//  LoopUITests
+//
+//  Created by Cameron Ingham on 2/2/24.
+//  Copyright © 2024 LoopKit Authors. All rights reserved.
+//
+
+import XCTest
+
+final class SystemSettingsScreen: BaseScreen {
+    
+    // MARK: Elements
+    
+    var loopCell: XCUIElement {
+        app.cells["Tidepool Loop"]
```

Comment:
Another Tidepool specific string in shared code.

Reply:
We can derive this. Will update.